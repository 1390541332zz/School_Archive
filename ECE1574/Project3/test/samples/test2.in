File: test/maps/blacksburg.asc
Greatest Inclination 6