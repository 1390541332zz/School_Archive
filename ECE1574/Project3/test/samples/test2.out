Loading: test/maps/blacksburg.asc
Number of rows: 13
Number of columns: 20
	         0    1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16   17   18   19
	    0  662  680  704  800  997  906  754  668  662  768  770  672  706  797  827  721  606  567  584  666
	    1  567  600  621  647  671  652  622  607  689  697  670  752  716  653  637  674  725  753  799  756
	    2  538  562  573  579  616  658  679  616  644  726  754  685  728  778  748  710  700  651  659  638
	    3  541  640  732  769  782  741  720  728  735  710  727  737  698  718  661  639  627  652  597  539
	    4  625  602  630  674  724  746  733  694  675  768  703  637  646  658  656  620  570  548  534  590
	    5  594  646  702  699  675  656  703  761  734  672  608  627  660  660  602  536  492  619  677  503
	    6  575  587  658  668  629  594  602  617  603  594  616  629  636  674  617  491  567  791  631  452
	    7  570  589  587  575  556  555  580  599  602  611  622  619  637  658  619  479  506  624  683  599
	    8  541  539  549  526  535  535  538  593  606  624  631  637  644  640  624  492  511  599  539  485
	    9  562  550  553  558  542  547  571  605  651  675  672  640  636  614  592  498  496  537  548  551
	   10  596  589  582  559  564  597  608  600  598  603  620  623  627  592  544  555  538  559  572  552
	   11  601  600  580  557  554  547  580  591  607  603  599  617  635  626  604  614  613  606  610  585
	   12  585  583  555  566  571  560  583  590  605  635  632  633  637  646  656  660  646  620  617  592
	
Command: Greatest Inclination
	Height change:  181
	Steps:          22
	Average change: 8.22727
	Starting row:   6
Number of rows: 13
Number of columns: 20
	     0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
	 0   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 1   -  -  -  -  -  -  -  -  -  -  -  \  -  -  -  -  >  >  >  X
	 2   -  -  -  -  -  -  -  -  -  -  /  -  >  >  >  /  -  -  -  -
	 3   -  -  -  -  \  \  -  -  -  -  ^  -  -  -  -  -  -  -  -  -
	 4   -  -  -  -  ^  ^  \  -  -  /  -  -  -  -  -  -  -  -  -  -
	 5   -  >  >  /  -  -  -  >  /  -  -  -  -  -  -  -  -  -  -  -
	 6   /  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 7   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 8   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	 9   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	10   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	11   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	12   -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -  -
	
